src/table/body-cell/td-element.tsx

```diff
@@ -90,6 +90,7 @@ export const TableTdElement = React.forwardRef<HTMLTableCellElement, TableTdElem
       collapseButtonLabel,
       verticalAlign,
       resizableColumns,
+      resizableStyle,
```

Comment:
The `resizableStyle` replaces `style` to make the cell API better predictable so that no unexpected attributes can be assigned from the outside.